### DISCUSSION TITLE
Expanded row view in locked view [Sep 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3745] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.92.4**  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/3517  
#  
> 🙋‍♂️ **alex.pro**  
#### What about edit? Or they are connected and it should start working when this issue is fixed?  
#  
> 🙋‍♂️ **alex.pro**  
#### Added a comment to the issue above  
#  
> 💁 **o1lab**  
#### its locked : so edit wont be there 🙂  
#  
> 🙋‍♂️ **alex.pro**  
#### hm... I thought it is locked for filters and so on  
#  
> 🙋‍♂️ **alex.pro**  
#### but records can be updated  
#  
> 🙋‍♂️ **alex.pro**  
#### like if you are editor you can edit records  
#  
> 🙋‍♂️ **alex.pro**  
#### but if you are viewer - you can't  
#  
> 🙋‍♂️ **alex.pro**  
#### Use case here: I set up filters, sorting, hide fields and no one can edit it  
#  
> 🙋‍♂️ **alex.pro**  
#### but every who has permissions can update records  
#  
> 🙋‍♂️ **alex.pro**  
#### what is the use case for everything is locked (without permissions dependancy)?  
#  
> 💁 **o1lab**  
#### will consider ur inputs  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016792118305497209)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
